ubuntudesign/maas.io
# Tidy base templates

- `base_index` and `base_docs` were basically the same. Remove `base_docs` and add extra work to `docs-base.tpl`.
- Remove tabs and fix indentation in `base_index`

QA
--

``` bash
make setup develop
```

Check the site looks the same as <http://maas.io>.
